bin/index.js

```diff
@@ -29,8 +30,11 @@ program
             return;
         }
 
-        require('../dashboard/server/index');
-    })
-    .parse(process.argv);
+        Server();
```

Comment:
Should be something like:
```suggestion
        Server.init();
```